doc/source/release-notes/Percona-XtraDB-Cluster-5.6.27-25.13.rst

```diff
@@ -0,0 +1,78 @@
+
+* `1370532 <https://bugs.launchpad.net/percona-xtradb-cluster/+bug/1370532>`_: Toggling ``wsrep_desync`` while node is paused is now blocked.
+
+* `1455098 <https://bugs.launchpad.net/percona-xtradb-cluster/+bug/1455098>`_: Fixed failure of LDI on partitioned table. This was caused by partitioned table handler disabling bin-logging and Native Handler (InnoDB) failing to generate needed bin-logs eventually causing skipping of statement replication.
+
+* `1503349 <https://bugs.launchpad.net/percona-xtradb-cluster/+bug/1503349>`_: ``garbd`` now uses default port number if it is not specified in ``sysconfig``.
+
+* `1505184 <https://bugs.launchpad.net/percona-xtradb-cluster/+bug/1505184>_`: Corrected ``wsrep_sst_auth`` script to ensure that user name and password for SST is passed to XtraBackup through internal command-line invocation. ``ps -ef`` doesn't list these credentials so passing it internally is fine, too.
+
+* `1520491 <https://bugs.launchpad.net/percona-xtradb-cluster/+bug/1520491>`_: ``FLUSH TABLE`` statements are not replicated any more, because it lead to an existing upstream fix pending deadlock error. This fix also takes care of original fix to avoid increment of local GTID.
+
+* Fixed **garb** init script causing new UUIDs to be generated every time it runs. This error was due to missing ``base_dir`` configuration when ``gardb`` didn't have write-access to current working directory. ``garbd`` will now try to use ``cwd``. Then it will try to use ``/var/lib/galera`` (like most Linux daemons). If it fails to use or create ``/var/lib/galera``, it will throw a fatal error.
+
+* Fixed replication of ``DROP TABLE`` statement with mix of temporary and non-temporary tables (``DROP TABLE temp_t1, non_temp_t2``), which caused errorneous ``DROP TEMPORARY TABLE stmt`` on replicated node. Corrected it by detecting such scenarios and creating temporary table on the replicated node, which is then dropped by follow-up ``DROP`` statement. All this workload should be part of same unit as temporary tables are session-specific.
+
+* Fixed ``wsrep_cluster_name`` value over 32 characters long to cause gmcast message to exceed maximum length. Imposed a limit of 32 character on ``wsrep_cluster_name``.
+
+* Added code to properly handle default values for ``wsrep_*`` variables, which used to cause an error/crash.
+
+* Fixed error when CTAS still tried to certify a transaction on a table without primary key even if certification of tables without primary key is disabled. This error was caused by CTAS setting ``trx_id`` (``fake_trx_id``) to execute ``SELECT`` and failing to reset it back to ``-1`` during ``INSERT`` as certification is disabled.
+
+* Fixed crashing of ``INSERT .... SELECT`` for MyISAM table with ``wsrep_replicate_myisam=ON``. This was caused by TOI being invoked twice when source and destination tables were MyISAM.
```

Comment:
table with :variable:`wsrep_replicate_myisam` set to ``ON`` would link to variable description